agent/org.linkedin.glu.agent-server-upgrade/src/cmdline/resources/bin/agentctl.sh

```diff
@@ -202,14 +204,19 @@ get_pid()
     PID=`cat $PID_FILE`
     PS_STAT=`ps -p $PID -o'user,pid=' | tail -1 | awk '{print $2}'`
 
-    case "$PS_STAT" in
-      $PID  ) echo $PID
-              ;;
-      *     ) echo 0
-              ;;
-    esac
+    if [[ "$PS_STAT" -eq "$PID" ]]; then
+      CMD=$(ps -p $PID -o 'comm=')
```

Comment:
Thanks for the pull request. Can you explain what those 2 lines do (208 and 209)?. I am not a shell expert :)